src/renderer/Components/Notes/Note.tsx

```diff
@@ -22,7 +22,9 @@ const Note = () => {
   const { note_id } = useParams();
 
   // Get space value stored in Redux Store.
-  const spaceState = useSelector((state: RootStateOrAny) => state.space);
+  const spaceState = useSelector(
```

Comment:
**TODO:** Update all `useSelector` and `useDispatch` with `useAppSelector` and `useAppDispatch` resp.